### PR TITLE
chore (refs DPLAN-12349): Add missing Hooks.

### PR DESCRIPTION
**Pull request by ahmad-demos**
### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-12349/Hooks-BOB-SH-Plattformeinstellungen

- Add missing Hooks.

### CODE REVIEW

File: client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue [hunk 1]
```diff
@@ -37,6 +37,7 @@
       <dp-input
         id="videoTitle"
         v-model="video.title"
+        data-cy="uploadedVideoTitle"
```
Comment (spiess-demos) [inline client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue:40]: to relate this to the surrounding section hooks i'd change it to

```suggestion
        data-cy="customerSettings:videoTitle"
```

File: client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue [hunk 2]
```diff
@@ -55,6 +56,7 @@
         needs-hidden-input
         required
         data-dp-validate-if="#videoTitle!=='', #videoDescription!==''"
+        data-cy="uploadVideo"
```
Comment (spiess-demos) [inline client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue:59]:
```suggestion
        data-cy="customerSettings:videoUpload"
```

And in line 66, the textarea needs a data-cy, too. I'd propose `customerSettings:videoDescription` as a value for it.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue (hunk 2 above)
Comment (ahmad-demos) [reply client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue:59]: Done in https://github.com/demos-europe/demosplan-core/pull/3632/commits/b41db82d5cc9234745a51ba2dc66fce5d9a2dede
Thnaks!

Comment (spiess-demos) [reply client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue:59]: Are you still planning to use the above suggestion?

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue (hunk 2 above)
Comment (ahmad-demos) [reply client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue]: Yes, done
thanks

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**